awx/api/serializers.py

```diff
@@ -4741,20 +4741,20 @@ def create(self, validated_data):
         WorkflowJobNode.objects.bulk_create(nodes)
 
         # Deal with the m2m objects we have to create once the node exists
-        for obj_type, obj_through_model in node_m2m_object_types_to_through_model.items():
-            through_models = []
+        for obj_type, through_model_obj in node_m2m_object_types_to_through_model.items():
```

Comment:
nitty nit pick - _this_ is not an object, but a class/model. The prior name of `obj_through_model` was more correct than this, although a little confused. Just `through_model` would be a better name.

Going a little off topic, but `obj_type` might more clearly named `field_name`. It is a field name, just the name of a ManyToManyField.

Reply:
ok I think I fixed it, thanks